code/modules/spells/cantrips.dm

```diff
@@ -0,0 +1,130 @@
+//cantrips: cheap fun spells
+
+//summon food: creates bread or a common fruit at target location
+/spell/aoe_turf/conjure/summon_food
+	name = "Feed The Multitude"
+	desc = "Creates a loaf of bread or a common fruit at the targeted location."
+
+	price = 0.2 * Sp_BASE_PRICE //1 point
```

Comment:
Pretty sure `round()` gets evaluated at compile time because it's a built in mathematical function so it should work.